ethereum/circuits/lib/src/account_with_storage_recursive_int_test.nr

```diff
@@ -0,0 +1,18 @@
+use crate::account_with_storage_recursive::{get_account_with_storage_recursive, RecursiveProof};
+use crate::account_with_storage::StorageWithinBlock;
+use crate::fixtures::mainnet::paris::usdc_circle::{header::{number, hash}, account::{address, account}, storage::{keys, values}};
+use crate::chain::ETHEREUM_MAINNET_ID;
+use crate::misc::types::{Bytes32, Address};
+use dep::std::test::OracleMock;
+use dep::std::unsafe::zeroed;
+
+#[test]
+fn success() {
+    let result = StorageWithinBlock { block_hash: hash, account, values };
+    // This is not a correct proof but it passes because nargo does not verify it. Nargo thinks that it's bb's job, but doesn't call bb.
+    let recursive_proof = RecursiveProof { proof: zeroed(), verification_key: zeroed(), key_hash: zeroed() };
```

Comment:
You can use zeroed on the whole struct once instead of using on each field

Reply:
done